### PR TITLE
adding dining preference dataset 

### PR DESCRIPTION
**Pull request by acostadon**
This dataset is very small, and uses strings as node names. It will be used to test force atlas, and in a new link prediction/similarity notebook.

the licensing is contained here.
http://networkdata.ics.uci.edu/netdata/html/Dining-table_partners.html

### CODE REVIEW

File: python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: dining_prefs
+file_type: .csv
+author: J.L. Moreno
```
Comment (nv-rliu) [inline python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml:3]: Between `file_type` and `author`, we should add a `description` field that briefly explains what this graph represents

File: python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+url:  https://data.rapids.ai/cugraph/datasets/dining_prefs.csv
```
Comment (nv-rliu) [inline python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml:4]: Move `url` field to the end of the file

File: python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - string
+  - string
+  - int
+has_loop: false
+is_directed: false
+is_multigraph: false
+is_symmetric: true
+number_of_edges: 42
+number_of_nodes: 26
+number_of_lines: 52
```
Comment (nv-rliu) [inline python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml:23]: We got rid of `number_of_lines` when promoting the experimental datasets to stable. I'm not entirely sure if we would keep this for certain datasets. Why is it necessary?

File: python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml (hunk 1 above)
Comment (BradReesWork) [reply python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml:3]: +1

Comment (acostadon) [reply python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml]: Done

File: python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml (hunk 3 above)
Comment (acostadon) [reply python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml]: Didn't know it wasn't necessary, noticed that the tests verify that the metadata is correct...which is great.
I removed the lines item.

File: python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml (hunk 2 above)
Comment (acostadon) [reply python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml]: moved to the end.

**Comment by acostadon** [comment]
rerun tests

**Review by BradReesWork — APPROVED** [review]

**Review by nv-rliu — APPROVED** [review]

**Pull request merged**